samples/voting/tye.yaml

```diff
@@ -1,4 +1,13 @@
 name: VotingSample
+ingress:
+  - name: ingress
+    bindings:
+      - port: 8080
+    rules:
+      - path: /vote(/|$)(.*)
```

Comment:
Did you test this?

Reply:
Yes, @davidfowl , I have tested this in my local Kubernetes cluster. Please find below details for your reference : 

![image](https://user-images.githubusercontent.com/13281246/98678044-f69b6880-2383-11eb-8b26-3f55a35e214e.png)

And then browsed the sample with 

![image](https://user-images.githubusercontent.com/13281246/98678103-0fa41980-2384-11eb-8459-e03b6326dd03.png)

Is there anything missing in the `tye.yaml`, which I should consider ?

Thanks.

Reply:
Did you run it locally with tye run?

Reply:
No, I used deploy option `tye deploy --interactive --namespace default` to deploy this sample in my local Kubernetes cluster. Also, please note, I had to use `namespace` parameter. Without that I was getting following error.

![image](https://user-images.githubusercontent.com/13281246/98686772-f8b6f480-238e-11eb-8c8a-b9659404b633.png)


`

Reply:
Can you double check this works with tye run locally as well?

Reply:
I have tested the sample with tye version `0.5.0-alpha` and it works fine in local.

Reply:
Not sure why the build fails on this PR. Any thoughts on that ?

Reply:
AFAIK we don't support regex like this in the rules in the local ingress

Reply:
I see, updated `tye.yaml`. Thank you for the clarification.